src/pykeen/models/__init__.py

```diff
@@ -93,10 +97,14 @@
 model_resolver = Resolver.from_subclasses(
     base=Model,
     skip={
+        # Abstract Models
         _NewAbstractModel,
+        EvaluationOnlyModel,
         # We might be able to relax this later
         ERModel,
         LiteralModel,
+        # baseline models behave differently
+        MarginalDistributionBaseline,
```

Comment:
@cthoyt I made them unavailable via model_resolver, since they cannot be used, e.g., with the pipeline,